### ISSUE TITLE
False positives for CSS `v-bind()` in `vue/no-extra-parens` rule

### ISSUE THREAD

**Issue by ota-meshi** [vuejs/eslint-plugin-vue#1774]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 8.7.0
- **eslint-plugin-vue version:** 8.3.0
- **Node version:** 16
- **Operating System:** Mac

**Please show your full configuration:**

```json5
{
 "vue/no-extra-parens": "error"
}
```

**What did you do?**	

```vue
<style>
a {
  color: v-bind(foo);
}
</style>
```

**What did you expect to happen?**

No error.

**What actually happened?**

Error.

```
[3:16]: Unnecessary parentheses around expression. ( vue/no-extra-parens )
```

**Repository to reproduce this issue**

[DEMO](https://ota-meshi.github.io/eslint-plugin-vue-demo/#eJw1yksKgCAQANCryKwKkqClSSdxYzpBIDPhj0K8e23aPl4Dxx5BgU75CbgZsqIZEsJx4KhElftJfjiYx9VQN6TnP8IEsQRMoBrUgjOxxDtHKy8bkT5een8BFeMg7w==)
